.travis.yml

```diff
@@ -20,7 +20,6 @@ script:
   - "test $GIMME_OS.$GIMME_ARCH != linux.amd64 || go generate"
   - "test $GIMME_OS.$GIMME_ARCH != linux.amd64 || git status"
   - "test $GIMME_OS.$GIMME_ARCH != linux.amd64 || test $(git status --porcelain | wc -l) == 0"
-  - "go get -ldflags \"-X main.revision=$(git rev-parse HEAD)\" -v -t ./..."
```

Comment:
We still need to `go install` otherwise releases won't work. So please change to `go install -ldflags ... -v ./...`.